media/css/m24/components/navigation-refresh.scss

```diff
@@ -221,16 +221,22 @@ $margin-top: 58px; // top margin offset for mobile navigation menu
         h2::after {
             @include bidi(((margin-left, $spacing-sm, margin-right, 0),));
             content: '';
-            background: url('/media/img/icons/caret-sprite.svg') no-repeat bottom left / auto 12px;
+            background: url('/media/protocol/img/icons/arrow-down.svg') no-repeat center center / auto 16px;
             display: inline-block;
             height: 12px;
             vertical-align: baseline;
             width: 12px;
+            transform-origin: center;
+            transition: transform 0.1s ease;
+
+            @media (prefers-reduced-motion: reduce) {
+                transition: none;
+            }
         }
 
         &.mzp-is-selected {
             h2::after {
-                background-position: bottom right;
+                transform: rotate(-180deg);
```

Comment:
Now I'm realizing the previous WMO nav had the rotation "on the inside" instead:

https://github.com/mozilla/bedrock/blob/2025-06-25/media/css/protocol/components/_menu.scss#L186

_(clockwise vs. counterclockwise)_

(Ideally that should get flipped the other direction for RTL but I admittedly skipped that here for simplicity.)

Is there a preference for one or another, or attempts at consistency even?

Reply:
It's the previous nav, I don't think there's any point trying to be consistent with it.